plugin.php

```diff
@@ -197,7 +197,7 @@ function wpcf7_metabox( $cf7 ) {
 				)
 			),
 			'extra-track-ga' => array(
-				'label' => __( 'Google Analytics Tracking', 'cf7-extras' ),
```

Comment:
@KZeni Maybe keep this as "Analytics Tracking" to better fit the two column layout?

Reply:
Agreed! Updated!